platform-docs-migration/README.md

```diff
@@ -0,0 +1,70 @@
+
+## State of the documentation
+
+Our [current documentation hub](https://docs.giantswarm.io) is an ongoing effort to maintain old Vintage entries that are no longer valid in our new product, and at the same time adequate to the new Dev Platform and CAPI concept.
+
+Today we have several work streams trying to achieve that goal:
+
+1) Horizon and SIG docs maintaining a coherent docs structure that follows new product developments.
+2) Teams updating doc entries for out-of-date documents. Most of the time they don`t have the time to rewrite the content to make it Platform/CAPI ready, so they stay out of date.
+3) Teams, KaaS and platform, adding new content that is only working for the new implementations (CAPI).
+
+## Goals
+
+The described scenario is not working and ideally we find an alternative that has these goals:
+
+- Have a good documentation portal that does not have entries out of date. Only new implementation (since we have completely deprecated Vintage).
+- Stop trying to bring current articles to life, most of them are no longer valid in the new implementation. Just add what is relevant and possibly identify spots to fill by teams.
+- Have a period for transition where everyone can look for content in Vintage and Dev Platform docs. Customers or support still have access to Vintage docs but new docs are also available.
+
+## Process
+
+Instead of creating a huge Pull Request in [our docs repository](https://github.com/giantswarm/docs) where we work in parallel (accumulating a lot of changes and configuration drifts), I propose to create a new clean space in our docs site. This new path is visible in as top level docs though the entry page still point to the old docs content. That way we can visualize the changes till we reach a point of satisfaction and furthermore, customers are migrated from Vintage product.
+
+**But why?**
+
+- In the new path we can create a structure according to the new necessities without having to transition from existing content.
+- We don`t need to maintain aliases or links to old entries or sections.
+- It is way easier to organize the work between teams to add needed minimal content for every section.
+- We can work in iterations.
+- Both docs sites will be publicly available.
+- The end result of our docs will be simpler and cleaner since we are not influenced by old structure and content.
+
+**How?**
+
+General steps to achieve the aforementioned goals:
+
+1) Moved the `vintage` content to a subpath so we leave a clean space in the top level docs site.
+2) Add aliases from old content entries to include the old path so we don`t break links.
+3) Leave the main docs entry page where it is now but pointing to the vintage path, so the appearance still the same.
+4) Avoid bots to scrape the docs for now on to avoid temporal indexes.
```

Comment:
```suggestion
4) Avoid bots to scrape the docs from now on to avoid temporal indexes. Add a banner to the vintage path to warn users of the state.
```